services/video-ingest.go

```diff
@@ -107,7 +107,13 @@ func getIngestIDFromURL(rtmpURL string) string {
 		return ""
 	}
 
-	ingestID := strings.Replace(parsed.Path, "/", "-", -1)
+	ingestID := parsed.Path
+
+	if strings.HasPrefix(ingestID, "/") {
```

Comment:
do you want to leave this check in, if TrimLeft is a no-op when there isn't a leading slash?

Reply:
We tested it yesterday, but adding test cases where we know there will be no preceding letter will be difficult.

Reply:
Potentially it could also help reduce the chance of bugs in the future?

Reply:
not sure you need to write a test for the builtin standard library

https://pkg.go.dev/strings#TrimLeft

Reply:
I think it's fine to not have a test for this as we know how to regression test against the `wasp-routing-service` that calls the `wasp-thing-service` for a thing lookup

Reply:
It would be quite a small test passing a test url to this function and expecting `stream-test` as a result. Might be worth doing as shouldn't take long but it's not a biggie really